engine/servermaster/jobop/backoff.go

```diff
@@ -0,0 +1,191 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jobop
```

Comment:
Jobop is somewhat confusing, please add some comments.

Reply:
Yes. A similar `jobop` in `CancelJob`.

Reply:
They will be in the same package actually, which contains high level job operations.

Reply:
added some comments